aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,52 @@
+import { Example } from '../../layouts';
+import { PageContentExample, ResponsiveContentLayoutExample } from '../../examples';
+
+**Page Component** is the main container that is responsible for how the Page Content and Page Header are 
+presented. 
+
+<Example 
+  showResponsiveControls={['fullScreen']}
+  docs="https://v2.grommet.io/page?theme=hpe"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js"
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+  screenContainer={{ scale: 0.57 }}
+>
+  <ResponsiveContentLayoutExample />
```

Comment:
Could we possibly change this example? When I look at it my focus is on the grid and it doesn't give me an idea of what Page and PageContent are doing. Maybe we could still include this example but not as the first one on the page.

Reply:
yeah maybe I can take out the cards and just keep the global header and footer then add some place holder text